src/utils/markets.tsx

```diff
@@ -330,6 +343,28 @@ export function _useUnfilteredTrades(limit = 10000) {
   //   .map(market.parseFillEvent.bind(market));
 }
 
+export function useBonfidaTrades() {
+  const { market } = useMarket();
+  async function getBonfidaTrades() {
+    if (!market) {
+      return null;
+    }
+    const response = await fetch(
+      `https://serum-api.bonfida.com/trades/address/${market.address.toBase58()}`,
```

Comment:
I would be in favor of adding a Bonfida api connector that does some of the boilerplate here for us. That way if we end up using more Bonfida sources we can avoid code replication.

Reply:
Done! See `bonfidaConnector.tsx`